src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,26 @@
+package fruitshop.strategy;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.Storage;
+
+public class ReturnOperation implements OperationHandler {
+    public static final String READ_FILE_PATH = "src/main/resources/reportToRead.csv";
```

Comment:
Avoid hardcoding file paths in your classes. Instead, pass the file path as a parameter to the method that needs it. This will make your code more flexible and easier to test.